Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:20.04
+
+ARG build_with_dependencies_source=0
+ARG sph_only_static_build=0
+ARG was_build=0
+ARG build_with_visualization=off
+
+ENV TZ=Europe/Berlin
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update && apt-get install -y \ 
+    apt-utils \
+    build-essential \
+    cmake \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN if [ "$build_with_visualization" = on ] ; then cd /home \
+    && apt-get update && apt-get install -y \ 
+    libglu1-mesa-dev freeglut3-dev mesa-common-dev libxi-dev libxmu-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*; fi
+
+RUN if [ "$build_with_dependencies_source" = 0 ] ; then cd /home \
+    && apt-get update && apt-get install -y \
+    libtbb-dev \
+    libboost-all-dev \
+    liblapack-dev \    
+    wget \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && wget https://github.com/simbody/simbody/archive/Simbody-3.7.tar.gz \
+    && tar xvzf Simbody-3.7.tar.gz \
+    && rm Simbody-3.7.tar.gz \
+    && mkdir /home/simbody-build && mkdir /home/simbody \
+    && cd /home/simbody-build \
+    && cmake /home/simbody-Simbody-3.7 -DCMAKE_INSTALL_PREFIX=$HOME/simbody -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_VISUALIZER=${build_with_visualization} -DBUILD_STATIC_LIBRARIES=on \
+    && make -j$(nproc) \
+    # && ctest -j$(nproc) \
```

Comment:
@Xiangyu-Hu  

Simbody failed 3 tests. Is it normal? 

215/216 Test #209: TestParallelForces ...................................   Passed    5.95 sec
216/216 Test #210: TestParallelForcesStatic .............................   Passed    5.92 sec

99% tests passed, 3 tests failed out of 216

Total Test time (real) =  16.10 sec

The following tests FAILED:
        170 - GazeboReactionForceWithAppliedForceCompliantStatic (Failed)
        172 - GazeboReactionForceWithAppliedForceRigidStatic (Failed)
        214 - TestReverseMobilizersStatic (Failed)

I commented out the test because of this failure